bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueClient.java

```diff
@@ -94,4 +94,12 @@
      * @param configUpdate the config update
      */
     void updateSensorConfig(FullSensor sensor, ConfigUpdate configUpdate);
+
+    /**
+     * Updates the given sensor.
+     *
+     * @param sensor the sensor to be updated
+     * @param stateUpdate the state update
+     */    
+	void updateSensorState(FullSensor sensor, StateUpdate stateUpdate);
```

Comment:
Please check the formatting. Take a look at the static-code-analysis report `target/code-analysis/report.html`

Reply:
OK, I will modify that in next commit